python/ml4ir/base/model/scoring/scoring_model.py

```diff
@@ -72,11 +72,11 @@ def final_activation_op(self, scores, metadata_features):
 
 class RelevanceScorer(ScorerBase):
     def architecture_op(self, train_features, metadata_features):
```

Comment:
Same here: need to add pydoc. What is an architecture_op?

Reply:
I have a work item to add pydocs for all methods. Will do it.